packages/flame/lib/src/collisions/hitboxes/screen_hitbox.dart

```diff
@@ -18,6 +18,9 @@ class ScreenHitbox<T extends FlameGame> extends PositionComponent
   @override
   void update(double dt) {
     super.update(dt);
+    // TODO(Lukas): Pass in a CameraComponent and use the position of the
```

Comment:
Issue to track?

Reply:
https://github.com/flame-engine/flame/issues/2544